core/src/main/java/edu/wpi/grip/core/OperationDescription.java

```diff
@@ -105,29 +105,20 @@ public boolean equals(@Nullable Object o) {
 
     OperationDescription that = (OperationDescription) o;
 
-    if (name() != null ? !name().equals(that.name()) : that.name() != null) {
-      return false;
-    }
-    if (summary() != null ? !summary().equals(that.summary()) : that.summary() != null) {
-      return false;
-    }
-    if (category() != that.category()) {
-      return false;
-    }
-    if (icon() != null ? !icon().equals(that.icon()) : that.icon() != null) {
-      return false;
-    }
-    return aliases() != null ? aliases().equals(that.aliases()) : that.aliases() == null;
-
+    return name().equals(that.name())
```

Comment:
Don't use the getter methods, just access the fields directly.

`name.equals(that.name)` can be replaced with `Objects.equals(name, that.name)`

This goes for lines 108 - 112